main.py

```diff
@@ -0,0 +1,37 @@
+class Printer:
+    instance = None
+
+    def __init__(self, model, type, isColor, isDuplex, paperTrayCapacity, paperCount):
```

Comment:
If I recall correctly, you need to make it possible to crete empty constructor (i. e. you need to pass default arguments to it)